xml/vk.xml

```diff
@@ -6828,7 +6828,7 @@ server.
                 <command name="vkQueueSignalReleaseImageANDROID"/>
             </require>
         </extension>
-        <extension name="VK_EXT_debug_report" number="12" type="instance" author="GOOGLE" contact="Courtney Goeltzenleuchter @courtney-g" supported="vulkan">
+        <extension name="VK_EXT_debug_report" number="12" type="instance" author="GOOGLE" contact="Courtney Goeltzenleuchter @courtney-g" supported="vulkan" obsoletedby="VK_EXT_debug_utils">
```

Comment:
We should change this one to "Promoted By" according to the new definitions.

Reply:
(I know there's some weirdness with "number of object handles returned" but given the old functionality is definitely a subset, it seems fine to say promotion - i.e. you could use this in the same way if you ignore extra objects)

Reply:
That is stretching the definition a lot.
- flags are split in two types
- `vkDebugReportMessageEXT` and `vkSubmitDebugUtilsMessageEXT` seems completely different
- `PFN_vkDebugReportCallbackEXT` and `PFN_vkDebugUtilsMessengerCallbackEXT` seem completelly different too

At the same time I don't see a harm as having it as `deprecatedby`. Promotion is just a more restricted type of deprecation.

Reply:
Yea I double checked what's changed and you're probably right. Let's call it deprecated.